configure.ac

```diff
@@ -42,9 +42,15 @@ AC_CANONICAL_HOST
 AC_CANONICAL_BUILD
 
 AS_CASE([${host_cpu}],
-        [aarch64], [RKT_ACI_ARCH=aarch64; GOARCH=arm64;],
-        [x86_64],  [RKT_ACI_ARCH=amd64; GOARCH=amd64;],
-        [AC_MSG_ERROR([*** unsupported host architecture specified: ${host_cpu}])])
+        [aarch64],             [RKT_ACI_ARCH=aarch64;     GOARCH=arm64;],
+        [armv7l],              [RKT_ACI_ARCH=unsupported; GOARCH=arm; GOARM=7;],
```

Comment:
I see what's the logic behind this (finding supported stage1 arch) but I find the use of `RKT_ACI_ARCH` for it quite confusing. I think this variable should instead mirror the labels at https://github.com/appc/spec/blob/master/schema/types/labels.go#L24 and the unsupported check should be self-contained in the `AS_VAR_IF` check later on.